scripts/extract-translations.js

```diff
@@ -11,9 +11,8 @@ const default_options = {
 };
 
 const parseCode = source => espree.parse(source, {
-    ecmaVersion: 10,
+    ecmaVersion: 'latest',
```

Comment:
hey @amir-deriv can you make this something static instead of latest?
asking this because in case of any update on ecma then it will use that, not that much safe

Reply:
Done @ali-hosseini-deriv 

![image](https://github.com/binary-com/SmartCharts/assets/129206554/af20d467-44f5-406a-982d-cfef04e12b70)

The error was happening due to optional chaining '?' which is part of `es2020` so the `ecmaVersion` of it is `11` , I have updated it to `11` the latest version is `13` right now